src/components/Map/Map.jsx

```diff
@@ -7,14 +7,17 @@ import "leaflet-defaulticon-compatibility";
 import {mapResults} from "@/components/Map/mockResults";
 import {ItemMarker} from "@/components/Map/ItemMarker";
 import L from "leaflet";
+import {SelectionOverlay} from "@/components/Map/SelectionOverlay";
+import {useChatMutation} from "@/store/services/chatApi";
 
 export const Map = (props) => {
   const {position: initialPosition} = props;
   const icon = L.icon({iconUrl: "/images/marker-icon.png"});
-
+  const [chat, { isLoading, isSuccess, data, error }] = useChatMutation();
```

Comment:
We should use /api/places endpoint here to get an answer like the constant data2 used.

Reply:
I've forgotten to remove `data2`, sorry